index.js

```diff
@@ -35,6 +35,12 @@ const editors = () => [{
 	paths: [],
 	keywords: []
 }, {
+	id: 'intellij',
+	name: 'IntelliJ',
+	bin: 'idea',
+	paths: [],
```

Comment:
The `idea` binary, or the app?
The app itself varies with version (mine is now `RUN_PATH = u'/Users/simen/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/171.4424.14/IntelliJ IDEA 2017.1 EAP.app'`).

The binary is generated by hand in the IDE, with a user-defined path. This is the default:

![image](https://cloud.githubusercontent.com/assets/1404810/25775997/6dcec332-32b2-11e7-83c4-26e57df3f86b.png)

Should I use that?

This is the exact same as webstorm, btw. WebStorm is just the JS parts of intellij, so changes to one here should change the other